Makefile

```diff
@@ -131,14 +131,24 @@ push: push-handler push-operator
 test/unit: $(GINKGO)
 	INTERFACES_FILTER="" NODE_NAME=node01 $(GINKGO) $(unit_test_args) $(WHAT)
 
-test/e2e: $(OPERATOR_SDK)
-	mkdir -p test_logs/e2e
-	unset GOFLAGS && $(OPERATOR_SDK) test local ./test/e2e \
+test-e2e-handler: $(OPERATOR_SDK)
+	mkdir -p test_logs/e2e/handler
+	unset GOFLAGS && $(OPERATOR_SDK) test local ./test/e2e/handler \
 		--kubeconfig $(KUBECONFIG) \
 		--namespace $(HANDLER_NAMESPACE) \
 		--no-setup \
 		--go-test-flags "$(e2e_test_args)"
 
+test-e2e-operator: $(OPERATOR_SDK)
+	mkdir -p test_logs/e2e/operator
+	unset GOFLAGS && $(OPERATOR_SDK) test local ./test/e2e/operator \
+		--kubeconfig $(KUBECONFIG) \
+		--namespace $(HANDLER_NAMESPACE) \
+		--no-setup \
+		--go-test-flags "$(e2e_test_args)"
+
```

Comment:
Maybe it's time to move this out of Makefile into hack/run-e2e-test.sh or the like so we continue reducing Makefile complexity.